semilattices.cabal

```diff
@@ -34,9 +34,9 @@ test-suite doctests
   main-is:             Doctests.hs
   default-language:    Haskell2010
   build-depends:       base
-                     , doctest
-                     , QuickCheck
-                     , quickcheck-instances
+                     , doctest >= 0.7 && < 1.0
+                     , QuickCheck >= 2.7 && < 2.12
+                     , quickcheck-instances == 0.3.*
```

Comment:
I guess this will make it easier for integrators to run the tests automatically, which seems like a plus! I hadn’t thought of that.